### PR TITLE
Core - new method to create service user

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Created new method createServiceUser which can be used to create a
service user without any members.

### CODE REVIEW

File: perun-base/src/main/resources/perun-roles.yml [hunk 1]
```diff
@@ -4306,6 +4306,11 @@ perun_policies:
     include_policies:
       - default_policy
 
+  createServiceUser_policy:
+    policy_roles:
```
Comment (balcirakpeter) [inline perun-base/src/main/resources/perun-roles.yml:4310]:
```suggestion
    policy_roles: []
```

**Review by balcirakpeter — APPROVED** [review]

File: perun-base/src/main/resources/perun-roles.yml (hunk 1 above)
Comment (Vojtech-Sassmann) [reply perun-base/src/main/resources/perun-roles.yml]: Fixed.

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java [hunk 2]
```diff
@@ -1422,6 +1424,64 @@ public void findUserByUserExtSourceLogin() {
...
+		User createdUser = usersManager.createServiceUser(sess, candidate, Collections.emptyList());
+
+		Attribute attr = perun.getAttributesManagerBl().getAttribute(sess, createdUser, URN_ATTR_USER_PREFERRED_MAIL);
+
+		assertThat(attr.getValue()).isEqualTo(value);
+	}
+
+	@Test
+	public void testCreateServiceUserSetsUes() throws Exception {
+		System.out.println(CLASS_NAME + "testCreateServiceUserSetsAttributes");
```
Comment (zlamalp) [inline perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java:1460]:
```suggestion
		System.out.println(CLASS_NAME + "testCreateServiceUserSetsUes");
```

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java [hunk 3]
```diff
@@ -1422,6 +1424,64 @@ public void findUserByUserExtSourceLogin() {
...
+		UserExtSource candidateUes = candidate.getUserExtSource();
+		User userByUes = usersManager.getUserByExtSourceNameAndExtLogin(sess, candidateUes.getExtSource().getName(),
+				candidateUes.getLogin());
+
+		assertThat(createdUser).isEqualByComparingTo(userByUes);
+	}
+
+	@Test
+	public void testCreateServiceUserFailsForAlreadyExistingUes() throws Exception {
+		System.out.println(CLASS_NAME + "testCreateServiceUserSetsAttributes");
```
Comment (zlamalp) [inline perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java:1475]:
```suggestion
		System.out.println(CLASS_NAME + "testCreateServiceUserFailsForAlreadyExistingUes");
```

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java (hunk 2 above)
Comment (Vojtech-Sassmann) [reply perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java]: Fixed.

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java (hunk 3 above)
Comment (Vojtech-Sassmann) [reply perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java]: Fixed.

**Review by zlamalp — APPROVED** [review]

**Pull request merged**